mmdet3d/ops/iou3d/iou3d_utils.py

```diff
@@ -21,34 +23,41 @@ def boxes_iou_bev(boxes_a, boxes_b):
 
 
 def nms_gpu(boxes, scores, thresh):
+    """Non maximum suppression on GPU.
+
+    Args:
+        boxes (torch.Tensor): Input boxes with shape (N, 5).
+        scores (torch.Tensor): Scores of predicted boxes with shape (N).
+        thresh (torch.Tensor): Threshold of non maximum suppression.
+
+    Returns:
+        torch.Tensor: Remaining indices with scores in descending order.
     """
-    :param boxes: (N, 5) [x1, y1, x2, y2, ry]
-    :param scores: (N)
-    :param thresh:
-    :return:
-    """
-    # areas = (x2 - x1) * (y2 - y1)
     order = scores.sort(0, descending=True)[1]
 
     boxes = boxes[order].contiguous()
 
     keep = torch.LongTensor(boxes.size(0))
```

Comment:
Should we also create keep by `boxes.new_zeros(boxes.size(0))` ?

Reply:
Yes. These ways to create new tensors could guarantee the normative access to specific cuda devices. (I tried to modify these parts before but I finally find the error is only caused by the cpp file. As a result I forget them in this PR...)